getting-started/installation-windows.rst

```diff
@@ -0,0 +1,116 @@
+﻿Installation on Windows
+=======================================
+
+What is EPICS about?
+-----------------------------------
+We assume that you know more or less what EPICS is. Here we want to start 
+from scratch and get to the point where we have a working server, then you 
+get on other how-tos to take you further. 
+
+Prepare your system
+-------------------
+
+You need ``make``, ``c++`` and ``libreadline`` to compile from source. 
+On Windows these dependencies can be installed by using msys2 tool. 
+This tool is available windows 7 onwards only. Currently this Installation is 
+tested only on windows 8.1. It should work for all the version of windows.
+In case we test it for other version, We will update the document.
+
+Install Tools
+-------------------
+MSYS2 provides a bash shell, Autotools, revision control systems and the like for building native Windows applications using MinGW-w64 toolchains. Tool can be installed from official website <https://www.msys2.org>. Download and run the installer - "x86_64" for 64-bit, "i686" for 32-bit Windows. Currently we go for 64 bit system. Installation procedure is well explained on website.
+
+Once instalaltion in complete, Open "MSYS2 MSYS" or "MSYS MinGW 64-bit". It shall provide you bash which resembles linux command shell. 
+Update MSYS2 with following command
+
+::
+
+    $ pacman -Syu
+  
+After finished Close the bash (do not exit). Open bash again and run the same command again to finish the updates.
+
+``tar`` is also needed to unpack the EPICS base
```

Comment:
I would like this document to preset 2 methods. 

1st is what I have written about. In which I install MYSYS2 bash and everything is done through that. Which gives Linux like the feel and all similar commands. In this method, you only need to follow the mentioned steps and use the mentioned tools and you should be good to go. This will be preferred by user who works generally works on Linux, but for special requirement has to use windows environment. 

2nd where we use windows interface. We can use strawberry or active Perl as a separate installation. We can use 7zip or WinZip. We have to modify some files manually. Install separate GNU makers like (Visual express studio, MinGW or Cygwin). THis is still not working flawlessly. IF I get this working properly, I intend to add a section for option 2.

Reply:
Just as a reference:
Have a look at the two `appveyor_*` scripts in [the .ci directory of base](https://github.com/epics-base/epics-base/tree/7.0/.ci). These are the scripts used on our Windows Continuous Integration service [AppVeyor](https://ci.appveyor.com/project/epics-base/epics-base) to compile Base using different Windows compilers. The `prepare` batch does necessary settings and installs prerequisites, the `make` batch configures and executes the build, using the appropriate setup batches supplied by the VS compilers. (These are different with almost every VS version...)
No Strawberries involved, though. (Perl is pre-installed on the AppVeyor build VMs.)

Reply:
You Mean, after installing MYSY2 if I run appveyor scrripts. Everything is done ? That sounds cool.

Reply:
No, no, no !
These AppVeyor scripts are `command.com` batch files used to compile Base using different Visual Studio and MinGW compilers without MSYS2 or Strawberry Perl. They can give you hints what you have to do to compile Base in such a setup.